IdentityCore/src/broker_operation/response/browser_native_message_response/MSIDBrowserNativeMessageGetTokenResponse.m

```diff
@@ -77,6 +77,11 @@ - (NSDictionary *)jsonDictionary
     response[@"account"] = accountJson;
     response[@"state"] = self.state;
     
+    __auto_type propertiesJson = [NSMutableDictionary new];
```

Comment:
Can this be a non-mutable dictionary?

Reply:
If we make it non-mutable, we will need to check for nullability of idTokenObj.username, I prefer to leave it mutable to simplify code.